src/AbstractNoUselessElseFixer.php

```diff
@@ -162,7 +162,7 @@ private function isInConditionWithoutBraces(Tokens $tokens, $index, $lowerLimitI
                 return true;
             }
 
-            if ($token->equals(';', '}')) {
```

Comment:
Didn't this meant to be `equalsAny`? Can we came with a test case for `}`?

Reply:
As `equals()` has only one argument, `'}'` was never used anyway so removing it has no impact on the behavior.